README.md

```diff
@@ -516,6 +516,119 @@ When passing parameters to the hook, be aware that arguments with spaces such as
 > In its regular mode, `nbstripout` acts as a filter and only modifies what git
 > gets to see for committing or diffing. The working copy stays intact.
 
+# nbstripout: guard PRs with github action
+
+`nbstripout` offers a re-usable GitHub action that verifies notebooks are properly stripped of output before merging. This is useful for enforcing clean notebooks in your CI/CD pipeline without modifying local files.
+
+## Basic usage
+
+```yaml
+name: Check Notebooks Output
+
+on:
+  pull_request:
+    paths:
+      - '**.ipynb'
+  push:
+    branches:
+      - main
+    paths:
+      - '**.ipynb'
+  workflow_dispatch:
```

Comment:
Is this meant to be empty? Is this valid syntax?

Reply:
Yes, `workflow_dispatch` just enables custom triggering of a workflow.